packages/KStepper/KStepState.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div
+    :class="{ 'pending': state === 'pending' }"
```

Comment:
question: what's the purpose of the `pending` class here?

Reply:
Removed